infrastructure/modules/cluster/main.tf

```diff
@@ -16,41 +16,23 @@ resource "azurerm_virtual_network" "cluster" {
   address_space       = [var.vnet_cidr]
 }
 
-data "azurerm_route_table" "load_balancer" {
```

Comment:
how come we're removing the load_balancer in this PR? description suggests it's just about creating a dev environment

Reply:
So this is the routing table for the load balancer which MHRA created for us. Here we're setting it up as a data source so that we can read info about its state. The dev environment needs to create its own routing table rather than read from one that already exists so I've moved this out of the `cluster` module into [`non-prod/main.tf`](https://github.com/MHRA/products/pull/571/files#diff-1da41d11e3dc7d7c623bc04b68d99563) (and `dev/main.tf`) and then just pass in the attributes that are needed as variables.